cloud-api/src/main/java/com/sequenceiq/cloudbreak/cloud/model/VmTypeMeta.java

```diff
@@ -93,6 +104,8 @@ public String toString() {
 
         private VolumeParameterConfig autoAttachedConfig;
 
+        private VolumeParameterConfig premiumSsd;
```

Comment:
Why we using premium here? Is Azure really provide Premium ssd or this is just a marketing name

Reply:
Partly a marketing name. Partly because Azure provides only one ssd right now but as I could tell, they will introduce an other type of ssd (ultra).

Reply:
I see. Then why we dont use ssd as premium and if the ultra ssd will come out then we will introduce this new field

Reply:
You're right, then I'm going to revert this part of the commit